compiler/src/dotty/tools/dotc/transform/AccessProxies.scala

```diff
@@ -70,6 +71,11 @@ abstract class AccessProxies {
     def accessorNameKind: ClassifiedNameKind
     def needsAccessor(sym: Symbol)(implicit ctx: Context): Boolean
 
+    def ifNoHost(reference: RefTree)(implicit ctx: Context): Tree = {
```

Comment:
This is overridden in both `ProtectedAccesors` and `Inliner`, shouldn't it be abstract?